remoteit/desktop
# [DESK-712] Added account avatar link and fixed label button spacing

### Description


Added account avatar link to settings page

### Test plan



1. Sign in
2. Go to settings page
3. Click the avatar in the top right
4. You should see the account page load in the browser

### Ticket



<https://remoteit.atlassian.net/browse/DESK-712>
